### ISSUE TITLE
Global Class

### ISSUE THREAD

**Issue by rodigu** [Liu-Hy/Food-Bank-Network-Simulation#2]
Would be useful to have a `Global` class with our constants and variables that need to be accessed throught the project (like days and food price).

**Comment by rodigu** [comment]
@lucianli123 @Liu-Hy

**Comment by Liu-Hy** [comment]
It makes sense

**Comment by rodigu** [comment]
Calendar class can be modified to fulfill the role partially.

Instead of the pantries calling on update calendar individually, we can have the simulation doing it.

**Comment by Liu-Hy** [comment]
Yes, using a global class is much better. I plan to remove calendar, after spending 1 or 2 days on a deadline.
